app/generic/proof/facades/acapy_proof_v1.py

```diff
@@ -0,0 +1,180 @@
+    IndyPresSpec,
+    IndyProofRequest,
+    V10PresentationCreateRequestRequest,
+    V10PresentationProblemReportRequest,
+    V10PresentationProposalRequest,
+    V10PresentationSendRequestRequest,
+)
+from aries_cloudcontroller.model.v10_presentation_exchange import (
+    V10PresentationExchange,
+)
+from fastapi.exceptions import HTTPException
+from pydantic.typing import NoneType
+
+from app.generic.proof.facades.acapy_proof import Proof
+from app.generic.proof.models import PresentationExchange
+
+logger = logging.getLogger(__name__)
+
+
+class ProofsV1(Proof):
+    @classmethod
+    async def create_proof_request(
+        cls,
+        controller: AcaPyClient,
+        proof_request: IndyProofRequest,
+        comment: str = None,
+        trace: bool = False,
+    ) -> V10PresentationExchange:
+        presentation_exchange = (
+            await controller.present_proof_v1_0.create_proof_request(
+                body=V10PresentationCreateRequestRequest(
+                    proof_request=proof_request,
+                    comment=comment,
+                    trace=trace,
+                )
+            )
+        )
+        return cls.__record_to_model(presentation_exchange)
+
+    @classmethod
+    async def send_proof_request(
+        cls,
+        controller: AcaPyClient,
+        proof_request: Union[
+            V10PresentationSendRequestRequest,
+            AdminAPIMessageTracing,
+            V10PresentationProposalRequest,
+        ],
+        free: bool = False,
+        pres_ex_id: str = None,
+    ) -> PresentationExchange:
+        try:
+            # This "free" is de facto the only one we support right now
+            if free:
+                presentation_exchange = (
+                    await controller.present_proof_v1_0.send_request_free(
+                        body=V10PresentationSendRequestRequest(**proof_request.dict())
+                    )
+                )
+            elif isinstance(proof_request, AdminAPIMessageTracing) and pres_ex_id:
+                presentation_exchange = (
+                    await controller.present_proof_v1_0.send_request(
+                        pres_ex_id=pres_ex_id, body=proof_request
+                    )
+                )
+            elif isinstance(proof_request, V10PresentationProposalRequest):
+                presentation_exchange = (
+                    await controller.present_proof_v1_0.send_proposal(
+                        body=proof_request
+                    )
+                )
+            else:
+                raise NotImplementedError
+            return cls.__record_to_model(presentation_exchange)
+        except Exception as e:
+            raise e from e
+
+    @classmethod
+    async def accept_proof_request(
+        cls,
+        controller: AcaPyClient,
+        pres_ex_id: str,
+        body: IndyPresSpec,
+    ) -> PresentationExchange:
+        pres_ex_id = cls.__pres_id_no_version(pres_ex_id=pres_ex_id)
```

Comment:
In the issuer implement the v1 prefix is removed in the issuer endpoints, but I  think how you handled it here  (in the acapy facades) makes more sense (the v1 implementation knows it should remove the `v1-` prefix). 

We should update the issuer implementation to be consistent with your method

Reply:
cool. I'll do that in this PR then.

Reply:
Will do in fidderent PR actually. There's too much in this one otherwise, imho